cluster/k8s-multus-1.10.4/README.md

```diff
@@ -0,0 +1,45 @@
+# Kubernetes 1.11.0 with multus network plugin in ephemeral containers
+
+Provides a pre-deployed Kubernetes with version 1.10.4 purely in docker
+containers with qemu. The provided VMs are completely ephemeral and are
+recreated on every cluster restart. The KubeVirt containers are built on the
+local machine and are the pushed to a registry which is exposed at
+`localhost:5000`.
+
+## Bringing the cluster up
+
+```bash
+export KUBEVIRT_PROVIDER=k8s-multus-1.11.0
+export KUBEVIRT_NUM_NODES=2 # master + one nodes
+make cluster-up
+```
+
+The cluster can be accessed as usual:
+
+```bash
+$ cluster/kubectl.sh get nodes
+NAME      STATUS     ROLES     AGE       VERSION
+node01    NotReady   master    31s       v1.11.0
+node02    NotReady   <none>    5s        v1.11.0
+```
+
+## Bringing the cluster down
+
+```bash
+export KUBEVIRT_PROVIDER=k8s-multus-1.11.0
```

Comment:
seems there is no k8s-multus-1.11.0 under kubevirt/cluster/

Reply:
Fix it.
Thanks!